Networking/Networking/Requests/RESTRequest.swift

```diff
@@ -50,7 +50,12 @@ struct RESTRequest: URLRequestConvertible {
         let components = [siteURL, Settings.basePath, wooApiVersion.path, path].map { $0.trimSlashes() }
         let url = try components.joined(separator: "/").asURL()
         let request = try URLRequest(url: url, method: method)
-        return try URLEncoding.default.encode(request, with: parameters)
+        switch method {
```

Comment:
nit: it'd be great if there's some documentation/reference on the encoding requirement, if any

Reply:
I agree. I found out about this just by comparing the requests from Android using a proxy tool. I couldn't find out any documentation about this. 😞